src/main/io/dashboard.h

```diff
@@ -43,7 +43,8 @@ typedef enum {
     PAGE_SENSORS,
     PAGE_RX,
     PAGE_PROFILE,
-#if defined(USE_TASK_STATISTICS)
+    PAGE_RPROF,
+#ifndef SKIP_TASK_STATISTICS
```

Comment:
For some reason you're using `SKIP_TASK_STATISTICS`, which has been replaced with `USE_TASK_STATISTICS` in latest master. I think this should be checked if some more rebasing has been missed.